libraries/manager/manager.py

```diff
@@ -605,6 +608,16 @@ def generate_dashboard(self, max_failures=MAX_FAILURES):
 
         return dashboard
 
+    def get_failures(self):
+        client = boto3.client('dynamodb')
+        return client.query( TableName=u'dev-tx-job',
+            IndexName = u'status-index',
+            Select     = u'ALL_PROJECTED_ATTRIBUTES',
+            KeyConditionExpression=u'#stts = :st',
+            ExpressionAttributeValues = {":st": {"S": "failed"}},
+            ExpressionAttributeNames = {"#stts": "status"},
```

Comment:
@bspidel If you change the index to have secondary sort key on created_at, and then add to the query ScanIndexForward = False, then the most recent failures will be read first.